tests/test_coref_model.py

```diff
@@ -18,3 +18,21 @@ def test_CoreferenceModel(nlp_da):  # noqa
     # test output format
     for output in outputs:
         assert isinstance(output, dict)
+
+
+def test_en_coref_model(nlp_en):  # noqa
+    model = CoreferenceModel(language="en")
+
+    text = [
+        "Luke Skywalker is from a moisture farm on Tatooine. "
+        "He flees when the Empire comes to find C-3PO and R2-D2.",
+    ]
+    docs = nlp_en.pipe(text)
+
+    # test batches forward
+    outputs = model.predict_batch_docs(docs)
+
+    # test output format
+    for output in outputs:
+        print(output)
+        assert isinstance(output, dict)
```

Comment:
Would you rather have it be something like:

```suggestion
    # create pipeline:
    nlp_en = ...
    nlp_en.add_pipe("my_coref_model") # either language should be guessed for passed in using config={...})

    text = [
        "Luke Skywalker is from a moisture farm on Tatooine. "
        "He flees when the Empire comes to find C-3PO and R2-D2.",
    ]
    docs = nlp_en.pipe(text)
    # test output format
    for doc in docs:
        output = doc.coreference_output
        assert isinstance(output, dict)
```

Reply:
I see what you're getting at, but no. This test is for the model, not the component. The way that it is set up now, the model and the component are too tightly integrated to my taste, but I think it is easier said than done to split it apart and does not give much value in the big picture.

The test is really just copy-paste from above with different text. :smile:

Reply:
Makes sense